database.py

```diff
@@ -9,9 +9,11 @@ class Map(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String)
+    image = db.Column(db.LargeBinary)
+    votes = db.Column(db.Integer)
```

Comment:
Default values should be added for this, otherwise the srb2 server will crash when trying to insert a new map.